src/MyBundle/EventListener/NoticePagerListener.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace MyBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+class NoticePagerListener
+{
+    /**
+     * @param FilterResponseEvent $event
+     */
+    public function onKernelResponse(FilterResponseEvent $event)
+    {
+        if ($event->getRequest()->get("_controller") === "mybundle.controller.job_controller:publishAction") {
```

Comment:
Ez jöhetne paraméterként, ne írd át, csak mondom, hogy a listenernek is lehet iNNYektálni, vagy legalább a controller service id-ját, vagy legalább változóként, és az action-t cseréled, de ez itt egy másik metódus amúgy is, mert le van másolva az if belseje, meg a $session is 2* van, meg az add notice is 2* van
mybundle.controller.job_controller:publishAction